src/views/Home/index.tsx

```diff
@@ -54,6 +58,9 @@ const HomeView = (): JSX.Element => {
   const [categories, setCategories] = useState<Categories | null>(null)
   const [activePrefab, setActivePrefab] = useState<IPrefab | null>(null)
   const [errorState, setErrorState] = useState<{ hasError: boolean, errorMessage: string }>(DEFAULT_ERROR_STATE)
+  const [selectedValues, setSelectedValues] = useState<OptionType[]>([]);
```

Comment:
Only a style thingy, I prefer doing `Array<OptionType>` instead of `OptionType[]`, we should probably pick one style :)